be/src/main/java/codesquad/issuetracker/domain/Assignee.java

```diff
@@ -34,4 +34,13 @@ public class Assignee {
     @JoinColumn(name = "member_id")
     @NotNull
     private Member member;
+
+    private Assignee(Issue issue, Member member) {
+        this.issue = issue;
+        this.member = member;
+    }
+
+    public static Assignee createAssignee(Issue issue, Member member) {
+        return new Assignee(issue, member);
+    }
```

Comment:
이펙티브 자바 아이템1 정적 팩토리 메소드

```suggestion
    public static Assignee of(Issue issue, Member member) {
        return new Assignee(issue, member);
    }
```

Reply:
이 리뷰를 보고 이펙티즈 자바 책을 주문했습니다..!! 😂

Reply:
다른 책들도 동일하겠지만 맹신하는건 좋지 않다고들 하더라구요.(특히 이펙티브자바가 광신도?들이 많나봅니다)
책을 너무 따르기보다는 코드를 보는 눈을 넓히는 목적으로 학습해야 하는듯 합니다.

저도 아직 부족하지만 혹시 이게 정답이다! 라고 여기실까봐 코멘트 드립니다 :)